lib/psabpf_table.c

```diff
@@ -105,14 +113,21 @@ static bool member_is_direct_object(psabpf_btf_t *btf, const struct btf_member *
     const struct btf_type *type = psabtf_get_type_by_id(btf->btf, member->type);
     if (type == NULL)
         return false;
-    if (!btf_is_struct(type))
-        return false;
     *type_name = btf__name_by_offset(btf->btf, type->name_off);
     if (*type_name == NULL)
         return false;
     if (strcmp(*type_name, "bpf_spin_lock") == 0)
         return false;
 
+    if (btf_is_int(type)) {
+        /* threat ActionSelector and ActionProfile references as a direct object */
```

Comment:
But `ActionSelector` and `ActionProfile` are not direct objects but table implementation attributes. I suggest to create separate method like `mebmer_is_implementation_attribute` or similar.

Reply:
Or rename `member_is_direct_object` to `..is_direct_or_implementation_object`

Reply:
Renamed